modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/attachment/AttachmentDDMFormFieldTemplateContextContributor.java

```diff
@@ -156,10 +173,10 @@ protected void activate(Map<String, Object> properties) {
 			ObjectConfiguration.class, properties);
 	}
 
-	private Object _getFileEntryProperties(
+	private <httpServletRequest> Object _getFileEntryProperties(
```

Comment:
This is wrong:
`private <httpServletRequest> Object _getFileEntryProperties(
`

It was not my intention to change the return of this method. I intended to add a new parameter `ThemeDisplay themeDisplay`.
See how it is [this part](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/attachment/AttachmentDDMFormFieldTemplateContextContributor.java#L159) in this class on master.

Reply:
Sorry about that. Can you fix it please?

Reply:
Don't worry. Thanks for noticing 😄